manage_breast_screening/assets/sass/main.scss

```diff
@@ -80,3 +80,49 @@ a[href="#"] {
     }
   }
 }
+
+.app-status-bar {
+  background-color: #00386e;
```

Comment:
I assume this scss is temporary but these colours and paddings should be aligned with the design system tokens where possible

Reply:
Is it worth extracting a var for it in that case, like $color_app-dark-blue? Or maybe using the colour functions of sass to adjust the shade of an existing colour?